architecture/pricing-sources-matrix.md

```diff
@@ -0,0 +1,75 @@
+# Kubecost Pricing Sources
+
+## Overview
+
+There are multiple ways that Kubecost can be configured to pull in pricing data. This document outlines the different options and how to configure them.
+
+## Pricing sources matrix
+
+* Note: with the exception of `Cloud Provider-Billing Integrated`, all pricing sources are configured per-cluster
+
+Kubecost supports the following pricing sources:
+
+| Source | Detail | Pros | Cons |
+|--|--|--|--|
+| [Cloud Provider OnDemand API](pricing-sources-matrix.md#cloud-provider-ondemand-api) | On by default<br>Kubecost looks up public pricing APIs. | No configuration<br>No maintenance | For users with significant discounts, Kubecost costs will be significantly higher than actual billing. |
+| [Custom Pricing](#custom-pricing) | Manually set monthly costs for: CPU, RAM, storage, GPU, network, spot CPU, spot RAM | Simple configuration<br>Does not require cloud provider access<br>Works for on-prem clusters<br>Completely air-gapped for highly-secure environments  | Accuracy dependent on value provided<br>Single rate per resource- no flexibility per node/storage types |
+| [Cloud Provider-Billing Integrated](/install-and-configure/install/cloud-integration/README.md) | Kubecost pulls cloud provider billing every 6 hours<br>When billing becomes available Kubecost reconciles the previously-estimated costs | Extremely accurate<br>Little maintenance<br>All cloud billing can be imported<br>Out-of-cluster costs can be [combined with Kubernetes resources](/using-kubecost/navigating-the-kubecost-ui/collections.md) | Kubecost requires access to the billing account<br>This setup can take time, especially if the team deploying Kubecost does not have access to the billing account |
+| [CSV Pricing](/install-and-configure/advanced-configuration/csv-pricing.md) | Kubecost Enterprise can use a user-provided CSV with granular resource prices. | Does not require cloud provider access<br>Works for on-prem clusters<br>Completely air-gapped for highly-secure environments | Mapping labels in the CSV can be tedious |
+| [AWS Spot Data Feed](/install-and-configure/install/cloud-integration/aws-cloud-integrations/aws-spot-instances.md) | Kubecost pulls spot prices from a custom feed | More accurate costs in the short-term (48 hours) for Spot nodes | Limited value if billing-integration is configured<br>Requires additional setup with AWS |
+| [Azure Rate Card](../install-and-configure/install/cloud-integration/azure-out-of-cluster/azure-config.md) | Kubecost pulls Azure Rate Card API | More accurate Azure costs in the short-term (48 hours) | Limited value if billing-integration is configured<br>Requires additional setup with Azure |
+| [Alibaba](../install-and-configure/install/provider-installations/alibaba-install.md) | Kubecost pulls Alibaba Rate Card API | Currently the only method to retrieve Alibaba resource pricing |More accurate costs<br>Requires additional setup with Alibaba |
```

Comment:
Suggest making multi-sentence answers either an unordered list (Can you do that in a table?) or adding proper punctuation to help delineate the details.

Reply:
I'll leave formattting to Brett.

Reply:
@bstuder99 what are your thougts on the bullets periods at end of sentences?